packages/velaux-ui/src/pages/Login/index.tsx

```diff
@@ -133,8 +155,12 @@ export default class LoginPage extends Component<Props, State> {
         span: 20,
       },
     };
-    const { loginType, loginErrorMessage, loginLoading } = this.state;
+    const { loginType, loginErrorMessage, loginLoading, firstTimeLogin } = this.state;
     const { Row, Col } = Grid;
+    let adminInfo: LoginUserInfo = {
+      name: "admin",
```

Comment:
Could we customize the administrator name?

Reply:
Yes we can always do, but let's see if anyone need that before do it.

Reply:
It's weird to just set a password. The user doesn't know what the username is.

Reply:
Well, that's true. Didn't see that happens. I'll make it configurable.

Reply:
First admin name can be customized now.